Docs/guidance_for_az_next.md

```diff
@@ -1,25 +1,25 @@
-## The guidance for az next
+## Instructions for installing and trying 'az next'
 
 ### How to install
 
-`az next` has been released as an extension, so you can install `az next` directly by executing `az extension add --source https://clirecommendation.blob.core.windows.net/package/next-0.1.0-py2.py3-none-any.whl --yes`.  *(If you have ever installed an older version of `az next`, please execute `az extension remove -n next` to uninstall the older version before installing the new version for `az next`)*
-
-However, you need to make sure that the version of `azure-cli-core` is 2.19.0+ (which can be viewed with `az version`). 
-If you are using an older version, please use `az upgrade` to upgrade.
+1. You need `azure-cli-core` version 2.19.0+. Run`az version`to check your version. Use `az upgrade` to upgrade.
+2. Install `az next` directly by executing `az extension add --source https://clirecommendation.blob.core.windows.net/package/next-0.1.0-py2.py3-none-any.whl --yes`.  
+  *If you have an older version of `az next`, run `az extension remove -n next` to uninstall the older version before installing the new version of `az next`*
+3. Run `az config set core.collect_telemetry=True` to turn on telemetry. This will help us improve az next recommendations based on your usage data
 
 ### How to play
 
-1. When you make an error in executing command, it will recommend the solution most used by others who has the same error. <br/>
-*Pelase note: This kind of recommendation need to turn on telemetry. If you haven not turned it on yet, please run `az config set core.collect_telemetry=True` and try again. If telemetry is not turned on, only this kind of recommendation will be affected, others can be used normally.*
+1. When you encounter an error, 'az next' will recommend resolutions, based on what other users used most frequently to resolve the same error. <br/>
+
 ![avatar](https://github.com/zhoxing-ms/image/blob/master/Screenshot%202021-01-06%20223136.png)
 
-2. When you don't know what to do next, it will recommend the next command to be executed most by others who has the same execution path
+2. When you don't know what to do next, 'az next' will recommend the next command to run, based on what other users with the same path used most frequently.
 ![avatar](https://github.com/zhoxing-ms/image/blob/master/Screenshot%202021-01-06%20223705.png)
 
-3. When you may be in a common E2E scenario, it will recommend the next set of command combinations in a specific scenario
+3. When you have a common E2E scenario, 'az next' will recommend the next set of command combinations, based on command combinations used most frequently by other users.
```

Comment:
In fact, the recommendation of scenario types comes from the knowledge base, and the data of these scenarios may be translated from the CLI scripts.

Reply:
ok maybe I didn't understand this. what is the knowledge base? your knowledge? or knowledge from CLI users?

this one is tricky.. I don't even think users would understand what E2E means in this case.. did we change this wording in the az next messaging itself? if yes, we could use the same wording here..

Reply:
This comes from our knowledge of manual input and the mining of CLI scripts provided by Aladdin

This comes from our manual input of usage knowledge and mining of the CLI scripts provided by Aladdin team.
If this implementation is not easy to describe, maybe we can simply tell users that we recommend some common E2E scenario command combinations.